### PR TITLE
[MAINTAIN-166] fixed issue with images on the blog post page for mobile screen

### LINKED ISSUE #2534: Carnation: Images on blog posts overflow the page on mobile
See https://sandbox-carnation-cus.openy.org/blog/mood-boosting-foods. I tested and this only applies to carnation.

Mood-boosting_foods___Drush_Site-Install

Something like this should do the trick:

```
.field-blog-image img {
    width: 100%;
    height: auto;
}
```

### PR DESCRIPTION
**Pull request by aleevas**
Original Issue, this PR is going to fix: [#2534](https://github.com/ymcatwincities/openy/issues/2534)

This one should fix an issue with image on blog post page that was overflowed the page on mobile
MAINTAIN-166fixedIssueWithImage

## Steps for review

- [ ] go to /blog/mood-boosting-foods
- [ ] switch to mobile screen
- [ ] verify that image looks good
- [ ] verify that image looks good on all types of screen

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by hamrant** [comment]
ok to test

**Review by hamrant — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3412/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3412/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3412/

**Comment by AnastasiiaPys** [comment]
retest please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3413/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3413/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3413/

**Comment by AnastasiiaPys** [comment]
Tested on macOs(Chrome), Samsung S10E, Iphone 11, Ipad, Iphone 5S
Looks good!
image

File: themes/openy_themes/openy_carnation/dist/main.js [hunk 1]
```diff
@@ -1791,7 +1791,7 @@
...
...{var i=this.tryEntries[e];if(i.tryLoc<=this.prev&&r.call(i,"finallyLoc")&&this.prev<i.finallyLoc){var o=i;break}}o&&("break"===t||"continue"===t)&&o.tryLoc<=n&&n<=o.finallyLoc&&(o=null);var u=o?o.completion:{};return u.type=t,u.arg=n,o?(this.method="next",this.next=o.finallyLoc,s):this.complete(u)},complete:function(t,n){if("throw"===t.type)throw t.arg;return"break"===t.type||"continue"===t.type?this.next=t.arg:"return"===t.type?(this.rval=this.arg=t.arg,this.method="return",this.next="end"):"normal"===t.type&&n&&(this.next=n),s},finish:function(t){for(var n=this.tryEntries.length-1;n>=0;--n){var r=this.tryEntries[n];if(r.finallyLoc===t)return this.complete(r.completion,r.afterLoc),S(r),s}},catch:function(t){for(var n=this.tryEntries.length-1;n>=0;--n){var r=this.tryEntries[n];if(r.tryLoc===t){var e=r.completion;if("throw"===e.type){var i=e.arg;S(r)}return i}}throw new Error("illegal catch attempt")},delegateYield:function(t,n,r){return this.delegate={iterator:E(t),resultName:n,nextLoc:r},"next"===this.method&&(this.arg=void 0),s}},t}(t.exports);try{regeneratorRuntime=e}catch(t){"object"==typeof globalThis?globalThis.regeneratorRuntime=e:Function("r","regeneratorRuntime = r")(e)}},
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/dist/main.js]: why js file is updated?
should we commit js source file change here? @aleevas

**Review by podarok — CHANGES_REQUESTED** [review]
js updated?

**Comment by podarok** [comment]
@aleevas - let's fix conflict, please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3487/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3487/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3487/

File: themes/openy_themes/openy_carnation/dist/main.js (hunk 1 above)
Comment (aleevas) [reply themes/openy_themes/openy_carnation/dist/main.js]: @podarok you're right, we don't need this. Was removed from the commit

**Comment by aleevas** [comment]
> @aleevas - let's fix conflict, please

@podarok the conflict was fixed in a new commit

**Pull request merged**